scripts/docker/debian/stretch/Dockerfile

```diff
@@ -105,5 +105,5 @@ WORKDIR /home/jenkins/elektra-cabal-sandbox
 RUN cabal update && \
 cabal sandbox init && \
 cabal install 'containers >=0.4 && <0.6' 'base >=4.7 && <5' 'directory >=1.1 && <1.4' \
-'process >=1.2 && <1.7' 'filepath >=1.3 && <1.5' 'base -any' 'hspec -any' 'QuickCheck -any' && \
-echo 'export HASKELL_SHARED_SANDBOX=/home/jenkins/elektra-cabal-sandbox' >> /etc/bash.bashrc
+'process >=1.2 && <1.7' 'filepath >=1.3 && <1.5' 'base -any' 'hspec -any' 'QuickCheck -any'
+ENV HASKELL_SHARED_SANDBOX /home/jenkins/elektra-cabal-sandbox
```

Comment:
You will need to set ENV before you try to use it.

Reply:
but i don't try to use it here it will be used later on by cmake then

Reply:
Ah so it is not a envvar for cabal?

Anyway your workdir instruction is lingering into the finished image so I would suggest you either remove it and set it via a RUN cd ... (faster) or add another WORKDIR instruction at the end (either to / as it was before or /home/jenkins)

Reply:
Also, you should probably use the ENV variable to navigate to that directory

Reply:
```
# Handle Haskell dependencies
ENV HASKELL_SHARED_SANDBOX /home/jenkins/elektra-cabal-sandbox
RUN mkdir -p ${HASKELL_SHARED_SANDBOX} \
    && cd ${HASKELL_SHARED_SANDBOX} \
    && cabal update && cabal sandbox init && cabal install ' \
           containers.....'
```

Reply:
no its a envvar for cmake so it uses this sandbox for the build process. Ok will update, thanks for the hints!